README.md

```diff
@@ -1,26 +1,26 @@
 <br>
 
 <div align="center">
-  <a href="https://hydralauncher.site">
-    <img src="./resources/icon.png" width="144"/>
-  </a>
+
+  [<img src="./resources/icon.png" width="144"/>](https://hydralauncher.site)
+
   <h1 align="center">Hydra Launcher</h1>
+  
   <p align="center">
     <strong>Hydra is a game launcher with its own embedded bittorrent client and a self-managed repack scraper.</strong>
   </p>
-  <p>
-    <a href="https://discord.gg/hydralauncher">
-      <img src ="https://img.shields.io/discord/1220692017311645737?style=flat&logo=discord&label=Hydra&labelColor=%231c1c1c"/>
-    </a>
-    <a href="https://github.com/hydralauncher/hydra">
-      <img src="https://img.shields.io/github/actions/workflow/status/hydralauncher/hydra/build.yml" />
-    </a>
-    <a href="https://github.com/hydralauncher/hydra">
-      <img src="https://img.shields.io/github/package-json/v/hydralauncher/hydra" />
-    </a>
-  </p>
 
-![Hydra Catalogue](./docs/screenshot.png)
+  [![build](https://img.shields.io/github/actions/workflow/status/hydralauncher/hydra/build.yml)](https://github.com/hydralauncher/hydra/actions)
+  [![release](https://img.shields.io/github/package-json/v/hydralauncher/hydra)](https://github.com/hydralauncher/hydra/releases)
+  
+  For this readme in other languages, choose here:
```

Comment:
I think this line is not needed. The tags are self-explanatory. What do you think?

![image](https://github.com/hydralauncher/hydra/assets/167933696/910e6c69-0169-4045-9d29-a5d35ba4fc8c)

Reply:
Yeaa, and then we dont need to update the "For this readme in other languages, choose here:" text in all languages, good point.

By me, we can remove this